ambry-frontend/src/test/java/com.github.ambry.frontend/AmbrySecurityServiceTest.java

```diff
@@ -679,6 +664,14 @@ private void verifyCacheHeaders(boolean isPrivate, MockRestResponseChannel restR
           restResponseChannel.getHeader(RestUtils.Headers.CACHE_CONTROL));
       Assert.assertNull("Pragma value should not have been set",
           restResponseChannel.getHeader(RestUtils.Headers.PRAGMA));
+    } else {
+      Assert.assertEquals("Expires value is incorrect for private blob",
```

Comment:
minor: can we fix the log statement. "incorrect for non-cachable blob"

Reply:
fixed